packages/expo-router/src/getServerManifest.ts

```diff
@@ -82,9 +82,11 @@ function uniqueBy<T>(arr: T[], key: (item: T) => string): T[] {
   });
 }
 
+type FlatNodeTuple = [contextKey: string, absoluteRoute: string, node: RouteNode];
```

Comment:
What do you think about using object here instead of array?

Then it will be far easier to use and debug when you do `x.contextKey` rather then `x[0]`

Reply:
Agree that an object would be nicer but I'd probably do that in a separate refactor, WDYT?